templates/openideal.template.yaml

```
@@ -0,0 +1,43 @@
+# The schema is versioned so that we can establish code paths differently in the future if we need to change this.
+version: 1
+
+# Templates are a small amount of information supporting a template URL. Each template is selectable at the project-creation step.
+info:
+  # Unique machine name, prefaced by a vendor or organization identifier
+  id: OpenideaL/OpenideaL.v3
```

```suggestion
  id: openideal/openideal
```

I think these have to be lowercase.  At least all of the others are.